src/ucs/time/timerq.h

```diff
@@ -11,6 +11,7 @@
 #include <ucs/time/time.h>
 #include <ucs/type/status.h>
 #include <ucs/sys/preprocessor.h>
+#include <ucs/type/spinlock.h>
 #include <pthread.h>
```

Comment:
do we still need `pthread.h`?

Reply:
`ucs_timer_queue_t` has member `ucs_spinlock_t lock;`
So I think this `#include` still need. (I'll rebuild it later without `#include`)

```C
typedef struct ucs_timer_queue {
    ucs_spinlock_t             lock;
    ucs_time_t                 min_interval; /* Expiration of next timer */
    ucs_timer_t                *timers;      /* Array of timers */
    unsigned                   num_timers;   /* Number of timers */
} ucs_timer_queue_t;
```

Reply:
Sorry, I was wrong. I'll remove `pthread.h`